MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,429 @@
+        
+        posterImage.snp.makeConstraints{
+            $0.edges.equalToSuperview()
+        }
+        
+        scrollView.snp.makeConstraints {
+            $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
+            $0.top.equalTo(posterView.snp.bottom)
+        }
+        
+        contentView.snp.makeConstraints {
+            $0.edges.equalTo(scrollView)
+            $0.width.equalTo(view)
+        }
+        
+        addConstraintsInPosterView()
+        addConstraintsInScroolView()
+    }
+    
+    func addConstraintsInPosterView() {
+        popularityLabel.snp.makeConstraints{
+            $0.leading.equalTo(posterView).inset(16)
+            $0.bottom.equalTo(titleLabel.snp.bottom).inset(28)
+        }
+        
+        userScoreLabel.snp.makeConstraints {
+            $0.leading.equalTo(popularityLabel.snp.trailing).offset(8)
+            $0.top.equalTo(popularityLabel.snp.top)
+        }
+        
+        titleLabel.snp.makeConstraints {
+            $0.leading.equalTo(popularityLabel.snp.leading)
+            $0.bottom.equalTo(releaseDateLabel.snp.bottom).inset(36)
+        }
+        
+        releaseYearLabel.snp.makeConstraints {
+            $0.leading.equalTo(titleLabel.snp.trailing).offset(8)
+            $0.bottom.equalTo(titleLabel.snp.bottom)
+        }
+        
+        releaseDateLabel.snp.makeConstraints {
+            $0.leading.equalTo(popularityLabel.snp.leading)
+            $0.bottom.equalTo(durationLabel.snp.bottom).inset(28)
+        }
+        
+        releaseCountryLabel.snp.makeConstraints {
+            $0.leading.equalTo(releaseDateLabel.snp.trailing).offset(8)
+            $0.top.equalTo(releaseDateLabel.snp.top)
+        }
+        
+        genreLabel.snp.makeConstraints {
+            $0.leading.equalTo(popularityLabel.snp.leading)
+            $0.bottom.equalTo(posterView.snp.bottom).inset(40)
+        }
+        
+        durationLabel.snp.makeConstraints {
+            $0.leading.equalTo(genreLabel.snp.trailing).offset(8)
+            $0.bottom.equalTo(genreLabel.snp.bottom)
+        }
+        
+        favoriteView.snp.makeConstraints {
+            $0.leading.equalTo(popularityLabel.snp.leading)
+            $0.top.equalTo(genreLabel.snp.bottom).offset(20)
+        }
+    }
+    
+    func addConstraintsInScroolView() {
+        
+        overviewLabel.snp.makeConstraints {
+            $0.leading.trailing.equalTo(contentView).offset(16)
+            $0.top.equalTo(contentView).offset(32)
+        }
+        
+        overviewTextLabel.snp.makeConstraints {
+            $0.leading.equalTo(overviewLabel.snp.leading)
+            $0.trailing.equalTo(contentView.snp.trailing).inset(16)
+            $0.top.equalTo(overviewLabel.snp.bottom).offset(20)
+        }
+        
+        caracter1.snp.makeConstraints {
+            $0.leading.equalTo(contentView).offset(16)
+            $0.top.equalTo(overviewTextLabel.snp.bottom).offset(32)
+            $0.width.equalTo(caracter2)
```

Comment:
We usually do not constrain the widths of the labels. I understand why you did it but we manage this as described in the first comment (stack view).